src/javaharness/.bazelrc

```diff
@@ -2,6 +2,9 @@
 # You can copy this into root of your workspace.
 
 build --incompatible_depset_union=false
+build --incompatible_disable_deprecated_attr_params=false
```

Comment:
What do these do?

Reply:
Newer versions of Skylark (BZL) rules and newer versions of Bazel barf on our build, because the J2CL compiler/Closure Compiler rules use deprecated bazel features.